### PR TITLE
fixed a flush(6)

### PR DESCRIPTION
**Pull request by komatits**

### CODE REVIEW

**Comment by buildbot-princeton** [comment]
The files changed require a buildbot test. Buildbot has started.

**Comment by buildbot-princeton** [comment]
Your changes passed the buildbot test. See [build 266](http://specfem3d-globe-buildbot.princeton.edu/builders/specfem3d_globe_builder/builds/266).

**Comment by komatits** [comment]
accept

On 12/13/2015 01:21 AM, buildbot-princeton wrote:
> Your changes passed the buildbot test. See build 266
> <http://specfem3d-globe-buildbot.princeton.edu/builders/specfem3d_globe_builder/builds/266>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/geodynamics/specfem3d_globe/pull/451#issuecomment-164204192>.
>

-- 
Dimitri Komatitsch
CNRS Research Director (DR CNRS), Laboratory of Mechanics and Acoustics,
UPR 7051, Marseille, France    http://komatitsch.free.fr

File: src/specfem3D/prepare_timerun.f90 [hunk 1]
```diff
@@ -3499,8 +3499,8 @@ subroutine make_inv_table(iphase,nglob,nspec,phase_nspec,phase_ispec,ibool,phase
       write(6,*) "   num_alloc_ibool_inv_tbl = ",num_alloc_ibool_inv_tbl
       write(6,*) "   num_used_ibool_inv_tbl  = ",num_used_ibool_inv_tbl
       write(6,*) "#### num_used_ibool_inv_tbl > num_alloc_ibool_inv_tbl  #####"
-      write(6,*) "#### Program sttoped ##########"
-      call flush(6)
+      write(6,*) "#### Program stopped ##########"
+      call flush_IMAIN()
```
Comment (QuLogic) [inline src/specfem3D/prepare_timerun.f90]: Shouldn't the above `write` calls be using `IMAIN` as well?

Comment (komatits) [inline src/specfem3D/prepare_timerun.f90]: You are right; let me change it.

On 12/13/2015 09:33 AM, Elliott Sales de Andrade wrote:
> In src/specfem3D/prepare_timerun.f90
> <https://github.com/geodynamics/specfem3d_globe/pull/451#discussion_r47441838>:
>
>> @@ -3499,8 +3499,8 @@ subroutine make_inv_table(iphase,nglob,nspec,phase_nspec,phase_ispec,ibool,phase
>>        write(6,*) "   num_alloc_ibool_inv_tbl = ",num_alloc_ibool_inv_tbl
>>        write(6,*) "   num_used_ibool_inv_tbl  = ",num_used_ibool_inv_tbl
>>        write(6,*) "#### num_used_ibool_inv_tbl > num_alloc_ibool_inv_tbl  #####"
>> -      write(6,*) "#### Program sttoped ##########"
>> -      call flush(6)
>> +      write(6,*) "#### Program stopped ##########"
>> +      call flush_IMAIN()
>
> Shouldn't the above |write| calls be using |IMAIN| as well?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/geodynamics/specfem3d_globe/pull/451/files#r47441838>.
>

-- 
Dimitri Komatitsch
CNRS Research Director (DR CNRS), Laboratory of Mechanics and Acoustics,
UPR 7051, Marseille, France    http://komatitsch.free.fr

**Pull request merged**